src/styles/components/message.scss

```diff
@@ -0,0 +1,54 @@
+@import '../colours';
+
+@function getMessageBorder($level) {
+  @return 0.25rem solid $level;
+}
+
+@mixin SVGDefinition($level) {
+  svg {
+    path {
+      fill: $level;
+    }
+  }
+}
+
+@mixin getWarningCSS($level) {
+  @include SVGDefinition(($level));
+  .alert__content {
+    border-left: getMessageBorder($level);
+  }
+}
+
+.alert {
+  -webkit-box-shadow: 0px 1px 3px 0px rgba(0, 0, 0, 0.2);
+  -moz-box-shadow: 0px 1px 3px 0px rgba(0, 0, 0, 0.2);
```

Comment:
Are those prefixes manually written or were they added by a tool? I'd say we needn't worry about those anymore. [see here for compatibility](https://caniuse.com/#feat=mdn-css_properties_box-shadow)

Reply:
Good point - now removed